### PR TITLE
KONFLUX-306: Add 4.14 cluster pool for Konflux

### PR DESCRIPTION
**Pull request by gbenhaim**

### CODE REVIEW

**Review by kasemAlem — APPROVED** [review]
/lgtm

File: clusters/hive/pools/rhtap/rhtap-ocp-4-14-0-amd64-aws-us-east-2_clusterpool.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  maxSize: 20
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: rhtap-aws-credentials
+      region: us-east-2
+  pullSecretRef:
+    name: pull-secret
+  runningCount: 1
+  size: 10
```
Comment (psturc) [inline clusters/hive/pools/rhtap/rhtap-ocp-4-14-0-amd64-aws-us-east-2_clusterpool.yaml:36]: IMO the size of a cluster pool **10** is too much for testing of a [single job](https://github.com/openshift/release/pull/48415/files#diff-142dad0885404df83d7c20686ac963d5b39a0a6e203fe6b0b338989d53506464R56). I'd start with 5. If it's too low, we can increase the size later. Wdyt?

Comment (gbenhaim) [reply clusters/hive/pools/rhtap/rhtap-ocp-4-14-0-amd64-aws-us-east-2_clusterpool.yaml]: I changed it to 5.

**Pull request merged**